### ISSUE TITLE
Segmentation fault: 11

### ISSUE THREAD

**Issue by ChrisTrivedi** [IGBB/quack#15]
Hi,

Really interested in trying out Quack over FastQC and Fastqp, but I'm getting an error after installation via `make && make test`. I seem to be able to call quack just fine, but then the only error I get is the segmentation error. I don't know anything about programming in C, and Googling such a common error hasn't really to provide me any info I might use to track down my issue.

Has anyone else had this error? Maybe I installed it wrong? I did get a slight error during the test, but I don't think this is the issue:

`../quack -1 ERR1438847_1.fastq.gz -2 ERR1438847_2.fastq.gz -a ../all.fa.gz -n ERR1438847 > paired.adapter.svg
convert paired.adapter.svg paired.adapter.png
make[1]: convert: No such file or directory
make[1]: *** [paired.adapter.png] Error 1
make: *** [images] Error 2`

I'm currently trying to run this on macOS Mojave 10.14.6. I also tried by calling the MacOS binary in /bin but that didn't work for me either. Any help would be greatly appreciated. Thanks!

**Comment by maa146** [comment]
You're correct about the error in `make test`. The ImageMagick command `convert` isn't installed. It's not an issue unless you want the images in something other than svg.

If memory serves, giving `quack` incorrect file paths -- for either of the raw data files or adapter file -- will make this happen. If the file paths are correct, can you give my the command you're trying to run?
